src/sql/src/statements/alter.rs

```diff
@@ -17,7 +17,7 @@ use sqlparser::ast::{ColumnDef, Ident, ObjectName, TableConstraint};
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct AlterTable {
     table_name: ObjectName,
-    alter_operation: AlterTableOperation,
+    pub alter_operation: AlterTableOperation,
```

Comment:
why change it to public and remove the getter? Not really prefer this way.

Reply:
`pub` the field to consume it here:

Reply:
I think the cost is really low here, and alter is not a frequent operation, so we don't need to concern about it.

Reply:
Changed back to private field, PTAL.